sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -923,7 +923,8 @@ glsTextureTestUtil.computeLodFromDerivates3D = function(mode, dudx, dvdx, dwdx,
             DE_ASSERT(false);
     }
 
-    return Math.log(p) * 1.44269504089;
+    // Native dEQP uses 32-bit numbers. So here 64-bit floating numbers should be transformed into 32-bit ones to ensure the correctness of the result.
+    return new Float32Array([Math.log(p)])[0] * new Float32Array([1.44269504089])[0];
```

Comment:
First, is this necessary? i.e., is it true without this conversion, the tests fail?

second, can you implement a helper function toFloat32 that uses a static FloatArray instead of create a new Float32Array for every conversion? We found this a perf bottleneck.